rust/warp/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "server"
-version = "0.1.0"
+version = "0.0.0"
```

Comment:
not sure to understand this change

Reply:
because this does not make sense for this web application, so making `0.0.0` better I think.

Reply:
agree, this information is not relevant. could not simply remove it ?

Reply:
I removed but build failed. So it looks like rust require version to be exists in order to build/compile.